cmd/kwild/config/config.go

```diff
@@ -119,6 +119,9 @@ type P2PConfig struct {
 	// PersistentPeers is a comma separated list of nodes to keep persistent
 	// connections to.
 	PersistentPeers string `mapstructure:"persistent_peers"`
+	// WhitelistPeers is a comma seperated list of nodeIDs that can connect to this node.
+	// This is excluding any persistent peers or seeds or current validators.
+	WhitelistPeers string `mapstructure:"whitelist_peers"`
```

Comment:
We probably want a config for running in private mode, right? Pretty much a `--private` flag that enforces this? The default action should still be public networks imo.